packages/story-editor/src/karma/fixture/fixture.js

```diff
@@ -230,7 +230,23 @@ export class Fixture {
     });
   }
 
-  restore() {
+  async restore() {
+    const styleSheets = window.document.querySelectorAll(
+      'link[rel=stylesheet]'
+    );
+
+    function clearSheets() {
```

Comment:
`removeEditorFontStyles` or something.

Reply:
absolutely, absolutely just a draft with minimal effort spent on clean up to focus on finding the root of the issues <3 <3

Reply:
so turns out once the links have proper ids they stop stacking. thanks for the add. I had just assumed it was intentionally left off, that's on me.